modules/network_synchronizer/networked_controller.h

```diff
@@ -514,17 +512,16 @@ struct DollController : public Controller {
 	NetUtility::StatisticalRingBuffer<uint32_t> network_watcher;
 
 	DollController(NetworkedController *p_node);
-	~DollController();
```

Comment:
This is necessary because the parent has a virtual destructor. We should add `virtual` and add the virtual destructor to the other classes too.